package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "kuzzle",
   "author": "The Kuzzle Team <support@kuzzle.io>",
-  "version": "2.17.6",
+  "version": "2.17.8",
```

Comment:
package-lock.json has not been updated?

Reply:
Right I just changed the version for my tests to be sure that I was not using the wrong version of Kuzzle